README.md

```diff
@@ -5,25 +5,42 @@ AWS also offers FreeRTOS Extended Maintenance Plan (EMP) that provides you with
 
 ## FreeRTOS/FreeRTOS Long Term Support
 
-Libraries in this GitHub branch (also listed below) are part of the [FreeRTOS 202012.05 LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012-LTS) patch release. Learn more at https://freertos.org/lts-libraries.html.
-
-| Library                     | Version             | LTS Until  | LTS Repo URL                                                                |
-|-------------------------    |---------------------|------------|---------------------------------------------------------------------------  |
-| FreeRTOS Kernel             | 10.4.3-LTS-Patch-3  | 03/31/2023 | https://github.com/FreeRTOS/FreeRTOS-Kernel/tree/V10.4.3-LTS-Patch-3        |
-| FreeRTOS-Plus-TCP           | 2.3.2-LTS-Patch-2   | 03/31/2023 | https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/tree/V2.3.2-LTS-Patch-2       |
-| coreMQTT                    | 1.1.0               | 03/31/2023 | https://github.com/FreeRTOS/coreMQTT/tree/v1.1.0                            |
-| coreHTTP                    | 2.0.0               | 03/31/2023 | https://github.com/FreeRTOS/coreHTTP/tree/v2.0.0                            |
-| corePKCS11                  | 3.0.0               | 03/31/2023 | https://github.com/FreeRTOS/corePKCS11/tree/v3.0.0                          |
-| coreJSON                    | 3.0.0               | 03/31/2023 | https://github.com/FreeRTOS/coreJSON/tree/v3.0.0                            |
-| backoffAlgorithm            | 1.0.0               | 03/31/2023 | https://github.com/FreeRTOS/backoffAlgorithm/tree/v1.0.0                    |
-| AWS IoT Device Shadow       | 1.0.2               | 03/31/2023 | https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/tree/v1.0.2   |
-| AWS IoT Device Defender     | 1.1.0               | 03/31/2023 | https://github.com/aws/Device-Defender-for-AWS-IoT-embedded-sdk/tree/v1.1.0 |
-| AWS IoT Jobs                | 1.1.0               | 03/31/2023 | https://github.com/aws/Jobs-for-AWS-IoT-embedded-sdk/tree/v1.1.0            |
-| AWS IoT Over-the-air Update | 3.0.0               | 03/31/2023 | https://github.com/aws/ota-for-aws-iot-embedded-sdk/tree/v3.0.0             |
+Libraries in this GitHub branch (also listed below) are part of the [FreeRTOS 202210-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202210-LTS) release. Learn more at https://freertos.org/lts-libraries.html.
+
```

Comment:
FreeRTOS 202210-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202210-LTS should be probably 202406

Reply:
Update the version and remove the link here. Thanks for the catch.